dashboard/public/frequency/frequency.js

```diff
@@ -1153,7 +1153,7 @@ $(document).ready(function () {
   $("#sort-toggle button").click(bg.handleSortChange.bind(bg));
 
   // When we switch back to shift mode, force an alphabetic order
-  $("a[href=#shift]").click(function () {
+  $("a[href='#shift']").click(function () {
     $("#sort-toggle button[value=alphabetic]").trigger('click');
```

Comment:
this should probably be quoted, too; `$("#sort-toggle button[value='alphabetic']")`